### PR TITLE
Stackoverflow tracking

### PR DESCRIPTION
**Pull request by erikakalar**
## What I Did
Added a Stackoverflow tracking code to Liquibase files. Added the call to the home page. 
## How To Test It
See if I did this right. Hopefully the tracking code will show up on the Home page so we'll know if our Stackoverflow marketing efforts are working. 

@tabuckner - Did I do this right? if yes, it'd be awesome to merge this into Staging and Master at your earliest convenience.

### CODE REVIEW

**Comment by erikakalar** [comment]
@tabuckner - you can ignore the quick hot fix banner thing. Nathan took care of that yesterday. My Git fu needs work.

File: _includes/tracking-codes/stackoverflow-tracking-code.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!-- Activity name for this tag: Datical Liquidbase Homepage Pixel -->
+<script type='text/javascript'>
+    var axel = Math.random()+"";
+    var a = axel * 10000000000000;
+    document.write('<img src="https://pubads.g.doubleclick.net/activity;xsp=4613943;ord='+ a +'?" width=1 height=1 border=0>');
+    </script>
+    <noscript>
+    <img src="https://pubads.g.doubleclick.net/activity;xsp=4613943;ord=1?" width=1 height=1 border=0>
```
Comment (tabuckner) [inline _includes/tracking-codes/stackoverflow-tracking-code.html]: i think we should double check this line. I believe we need quotes, and i'm not sure Border is a valid property for the image tag. Just a quick double check is good enough :)

File: _includes/tracking-codes/stackoverflow-tracking-code.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!-- Activity name for this tag: Datical Liquidbase Homepage Pixel -->
+<script type='text/javascript'>
+    var axel = Math.random()+"";
```
[the remark this answers is not in the document]
Comment (erikakalar) [reply _includes/tracking-codes/stackoverflow-tracking-code.html]: crazy. yeah. that's exactly what they gave me. LOL.

File: _includes/tracking-codes/stackoverflow-tracking-code.html (hunk 1 above)
Comment (erikakalar) [reply _includes/tracking-codes/stackoverflow-tracking-code.html]: Okay! how do we double check? anything specific I need to do?

**Pull request merged**